operator/controllers/seldondeployment_controller.go

```diff
@@ -170,6 +170,46 @@ func createIstioResources(mlDep *machinelearningv1.SeldonDeployment,
 			return nil, nil, err
 		}
 	}
+
+	getDefaultHttpRoute := func() istio_networking.HTTPRoute {
+		route := istio_networking.HTTPRoute{
+			Match: []*istio_networking.HTTPMatchRequest{
```

Comment:
Why should we keep a pointer slice here?

Reply:
The point slice is definition by istio